docs/plugins/filters/dns.asciidoc

```diff
@@ -127,10 +127,28 @@ number of times to retry a failed resolve/reverse
 [id="plugins-{type}s-{plugin}-nameserver"]
 ===== `nameserver` 
 
-  * Value type is <<array,array>>
+  * Value type is <<hash,hash>>, and is composed of:
+    * a required `address` key, whose value is either a <<string,string>> or an <<array,array>>, representing one or more nameserver ip addresses
+    * an optional `search` key, whose value is either a <<string,string>> or an <<array,array>>, representing between one and six search domains (e.g., with search domain `com`, a query for `example` will match DNS entries for `example.com`)
+    * an optional `ndots` key, used in conjunction with `search`, whose value is a <<number,number>>, representing the minimum number of dots in a domain name being resolved that will _prevent_ the search domains from being used (default `1`; this option is rarely needed)
+  * For backward-compatibility, values of <<string,string>> and <<array,array>> are also accepted, representing one or more nameserver ip addresses _without_ search domains.
```

Comment:
This is not rendering as intended. 
<img width="964" alt="Screen Shot 2019-10-23 at 5 21 01 PM" src="https://user-images.githubusercontent.com/35154725/67435176-95a6c400-f5b9-11e9-9707-4654cd5ccac3.png">
I will fix formatting and commit.